molecule/default/prepare.yml

```diff
@@ -1,29 +1,71 @@
 ---
-# The Ubuntu Docker images we are using
-# (geerlingguy/docker-ubuntu1604-ansible:latest and
-# geerlingguy/docker-ubuntu1804-ansible:latest) require a cache update
-# before they can install anything.
+- name: Import upgrade playbook
+  import_playbook: upgrade.yml
+
+- name: Import python playbook
+  import_playbook: python.yml
+
+# We want to copy a cron job to /etc/cron.daily, so that directory
+# needs to exist.  In our Kali and Debian 11 images it does not, so we
+# need to install a cron implementation.  On these platforms, the
+# default cron package name is called cron.
+#
+# Debian and Kali ship with an AppArmor profile for /usr/bin/freshclam
+# that, when applied to /usr/bin/freshclam running in a Docker
+# container, does not allow /usr/bin/freshclam to query DNS.  (The
+# same profile works as expected when run outside of Docker on a
+# Debian or Kali AMI.)  This obviously interferes with the downloading
+# of updated ClamAV databases and breaks this Ansible role.  Until we
+# can figure out exactly what is happening, the only remedy I have
+# found is to switch the AppArmor profile for /usr/bin/freshclam to
+# complain mode (or disable it entirely).
```

Comment:
We should probably create an issue for this and link to it in this comment for posterity.

Reply:
See #16.  The source code links in the body of the issue should be correct once this PR is merged.